### PR TITLE
Improvement/code cleanup

### PR DESCRIPTION
**Pull request by vquelque**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sweng/eventmanager/repository/EventRepository.java [hunk 1]
```diff
@@ -38,6 +38,6 @@
      */
     Task<Event> updateEvent(Event event);
 
-    Task deleteEvent(Event event);
+    Task<Event> deleteEvent(Event event);
```
Comment (zyuiop) [inline app/src/main/java/ch/epfl/sweng/eventmanager/repository/EventRepository.java:41]: why tho, Task<Void> would be better I think

Comment (vquelque) [reply app/src/main/java/ch/epfl/sweng/eventmanager/repository/EventRepository.java:41]: Yes you're right

Comment (zyuiop) [reply app/src/main/java/ch/epfl/sweng/eventmanager/repository/EventRepository.java:41]: `Task<Void>` I meant

**Review by zyuiop — APPROVED** [review]

File: app/src/main/java/ch/epfl/sweng/eventmanager/repository/EventRepository.java (hunk 1 above)
Comment (vquelque) [reply app/src/main/java/ch/epfl/sweng/eventmanager/repository/EventRepository.java:41]: Ahh yes! It's better for sure! I'll fix this

**Pull request merged**